### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by ChristianVonMach**
First attempt to solve #8.

### CODE REVIEW

**Comment by ChristianVonMach** [comment]
This implements one unit test for the Field::swap method in src/field/Field.cpp as first unit test all other unit tests should be similar.

Open Questions:
  - do we make a test target for every Class oder on big test for all classes?
  - do we want a additional gpu / multicor unit test? (in respect to question 1)
  - How do we add a simpler global config for every test?

**Comment by LinhWuerzburger** [comment]
> This implements one unit test for the Field::swap method in src/field/Field.cpp as first unit test all other unit tests should be similar.
> 
> Open Questions:
> 
>     * do we make a test target for every Class oder on big test for all classes?
> 
>     * do we want a additional gpu / multicor unit test? (in respect to question 1)
> 
>     * How do we add a simpler global config for every test?

* one test class for each class (to test)
* there may be cases where it makes sense to do some additional GPU tests. The multicore version is just a by-product, so we shouldn't focus on that.
* I have no idea.

File: gtests/field/Field.cpp
Comment (LinhWuerzburger) [inline gtests/field/Field.cpp:16]: we wanted to go with underscores and not camel case

**Review by LinhWuerzburger — APPROVED** [review]

**Comment by LinhWuerzburger** [comment]
btw, I linked the PR to the issue but it is not like we want to close the issue. But it should definitely be linked to the issue. I propose to reopen the issue after it got closed through this PR approval?

**Pull request merged**
